docs/flexSearch.md

```diff
@@ -132,25 +136,25 @@ To prevent slow queries, cruddl returns an error in these cases.
 
 The following filter-fields are available for fields that are annotated with `@flexSearch`:
 
--   `equals`
--   `not_equals`
--   `in`
--   `not_in`
--   `starts_with`
--   `not_starts_with`
+- `equals`
+- `not_equals`
+- `in`
+- `not_in`
+- `starts_with`
+- `not_starts_with`
```

Comment:
these changes are weird.  Can you check if you have set up prettier correctly?

Reply:
apparently my webstorm only does "apply prettier on auto-format" for javascript and typescript files. Not for markdown.